### DISCUSSION TITLE
Welcome to node-oracledb 6.0 which is now 'Thin' by default

### DISCUSSION THREAD

**Discussion by cjbj** [oracle/node-oracledb#1551] [Announcements]
We have released node-oracledb 6.0 !

The release announcement is [here](https://medium.com/@sharad-chandran/usher-in-a-new-era-with-the-node-oracledb-6-0-pure-javascript-thin-driver-e10e2af693b2).

By default, node-oracledb now runs in a 'Thin' mode which connects directly to Oracle Database. This mode does not need Oracle Client libraries. However, when the driver does use these libraries to communicate to Oracle Database, then node-oracledb is said to be in 'Thick' mode and has [additional functionality](https://node-oracledb.readthedocs.io/en/latest/user_guide/initialization.html#enabling-node-oracledb-thick-mode) available.

**Comment by yasirharis** [comment]
@cjbj will try this in couple minutes. Good jobs

**Comment by Blira** [comment]
Excellent, I've been waiting for this for years! Thank you very much!

**Reply by krismohan** [reply]
Let us know your application architecture. How much ever you can describe! Is it a standalone application that connects to the database? A single connection, or a pool of connections? Does it serve as a web server taking concurrent requests?
